templates/commands/render_action.go

```diff
@@ -64,6 +72,10 @@ func walkAndModify(ctx context.Context, pos *model.ConfigPos, rfs renderFS, scra
 			return nil
 		}
 
+		if _, ok := seen[path]; ok {
```

Comment:
I'm not sure, but I wonder if we should call filepath.Clean() (https://pkg.go.dev/path/filepath#Clean) on this to make sure we're seeing a normalized version of the filename.

Suppose somebody does this weird thing:

```
action: 'include'
paths: ['a.txt', './a.txt']
```

We'd want to detect that `mydir/a.txt` and `./mydir/a.txt` are the same file.

In fact, we can probably put the call to filepath.Clean() in `safeRelPath()` and fix this entire class of bugs forever.

Reply:
I put in a test and it seems to already be handled correctly, do you think further modification is needed?

Reply:
No need. I think what we're seeing is that filepath.Join() already calls clean on its output, so we're already getting the behavior we wanted. Thanks for checking.